packages/scheduler/src/forks/SchedulerHostConfig.default.js

```diff
@@ -359,50 +351,8 @@ if (
         // Start a rAF loop.
         isRAFLoopRunning = true;
         requestAnimationFrame(rAFTime => {
-          if (requestIdleCallbackBeforeFirstFrame) {
-            cancelIdleCallback(idleCallbackID);
-          }
-          if (requestTimerEventBeforeFirstFrame) {
-            clearTimeout(idleTimeoutID);
-          }
           onAnimationFrame(rAFTime);
         });
-
-        // If we just missed the last vsync, the next rAF might not happen for
-        // another frame. To claim as much idle time as possible, post a
-        // callback with `requestIdleCallback`, which should fire if there's
-        // idle time left in the frame.
-        //
-        // This should only be an issue for the first rAF in the loop;
-        // subsequent rAFs are scheduled at the beginning of the
-        // preceding frame.
-        let idleCallbackID;
-        if (requestIdleCallbackBeforeFirstFrame) {
-          idleCallbackID = requestIdleCallback(
-            function onIdleCallbackBeforeFirstFrame() {
-              if (requestTimerEventBeforeFirstFrame) {
-                clearTimeout(idleTimeoutID);
-              }
-              frameDeadline = getCurrentTime() + frameLength;
-              performWorkUntilDeadline();
-            },
-          );
-        }
-        // Alternate strategy to address the same problem. Scheduler a timer
-        // with no delay. If this fires before the rAF, that likely indicates
-        // that there's idle time before the next vsync. This isn't always the
-        // case, but we'll be aggressive and assume it is, as a trade off to
-        // prevent idle periods.
-        let idleTimeoutID;
-        if (requestTimerEventBeforeFirstFrame) {
-          idleTimeoutID = setTimeout(function onTimerEventBeforeFirstFrame() {
```

Comment:
Since you've removed this timeout, there's now no timeout fallback for the initial `rAF` call, like there is for subsequent rAFs: https://github.com/facebook/react/blob/ff006451ad792ed26793691402bf7b5f813205f7/packages/scheduler/src/forks/SchedulerHostConfig.default.js#L306-L311

But it's whatever since we're going to disable the rAF stuff anyway.

Reply:
That timeout should have already been in the `else` branch of this feature, anyway.

Reply:
>there's now no timeout fallback for the initial rAF call, like there is for subsequent rAFs

I guess that explains https://github.com/facebook/react/issues/16629?